web/src/features/hfiCalculator/components/LastUpdatedHeader.tsx

```diff
@@ -54,23 +55,22 @@ const findLastUpdate = (dailies?: StationDaily[]) => {
 }
 
 const LastUpdatedHeader = (props: LastUpdatedHeaderProps) => {
-  const classes = useStyles()
   const lastUpdate = findLastUpdate(props.dailies)
   if (lastUpdate) {
     const dateString = lastUpdate.toFormat('MMMM d, HH:mm') + ' PST'
 
     return (
       <StyledEngineProvider injectFirst>
         <ThemeProvider theme={lastUpdatedTheme}>
-          <span className={classes.container}>
+          <Container>
             <UpdateIcon></UpdateIcon>
-            <p className={classes.headerText}>Forecast last updated {dateString}</p>
-          </span>
+            <HeaderText>Forecast last updated {dateString}</HeaderText>
+          </Container>
         </ThemeProvider>
       </StyledEngineProvider>
     )
   } else {
-    return <React.Fragment></React.Fragment>
+    return <></>
```

Comment:
Other empty tags have been changed to divs. Consider this here for the sake of consistency.
```suggestion
    return <div></div>
```

Reply:
Changed back, see: https://github.com/bcgov/wps/pull/2998#discussion_r1265905205